pages/index.tsx

```diff
@@ -1,15 +1,9 @@
-import Head from 'next/head';
-import { HomeView } from '../routing/index';
-
 export default function Home() {
   return (
-    <div>
-      <Head>
-        <title>KaChing Cash Register Admin Panel</title>
-        <meta name="description" content="" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-      <HomeView />
-    </div>
+    <>
```

Comment:
why `<h1>Hello World</h1>`, what's the point of this page?

Reply:
done